src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -480,19 +480,19 @@ function tryRemoveElement(elem) {
                     secondaryTrackValid = false;
                 }
                 // secondary track should not be shown if primary track is no longer a valid pair
-                if (initialSubtitleStream && !playbackManager.trackHasSecondarySubtitleSupport(initialSubtitleStream)) {
+                if (initialSubtitleStream && !playbackManager.trackHasSecondarySubtitleSupport(initialSubtitleStream, this)) {
                     secondaryTrackValid = false;
                 }
             } else {
                 secondaryTrackValid = false;
             }
 
             // Get the secondary track that has been set during this watch session
-            let currentSecondaryTrackIndex = playbackManager.getSecondarySubtitleStreamIndex();
+            let currentSecondaryTrackIndex = playbackManager.getSecondarySubtitleStreamIndex(this);
 
             if (!secondaryTrackValid) {
                 currentSecondaryTrackIndex = -1;
-                playbackManager.setSecondarySubtitleStreamIndex(currentSecondaryTrackIndex);
```

Comment:
@thornbill @dmitrylyzo to update regarding that regression mentioned earlier, it was caused by these calls to `playbackManager`.

They were originally using the default fallback player defined in those methods, but I guess the timing has now changed. These were coming up as `undefined` in the methods, but they were never actually entering into the logic that throws an error for some reason. 
```
if (!player) {
    throw new Error('player cannot be null');
}
```

Maybe there was some sort of race condition where the HLS player would start before it reached that point? I'm not sure.

Either way, the fix was to just pass `this` into those methods, and it is now working as it should.

![SCR-20230210-jek](https://user-images.githubusercontent.com/30599893/218175319-26767881-b838-4d47-ab10-9efeb366b3a7.png)